CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Bottom level categories:
 
 - Change `AdapterInfo::{device,vendor}` to be `u32` instead of `usize`. By @ameknite in [#3760](https://github.com/gfx-rs/wgpu/pull/3760)
 - Remove the `backend_bits` parameter in `initialize_adapter_from_env` and `initialize_adapter_from_env_or_default` - use [InstanceDescriptor::backends](https://docs.rs/wgpu/latest/wgpu/struct.InstanceDescriptor.html#structfield.backends) instead. By @fornwall in [#3904](https://github.com/gfx-rs/wgpu/pull/3904)
+- Add a `compatible_surface` parameter to `initialize_adapter_from_env` and use that to make `initialize_adapter_from_env_or_default` always respect its `compatible_surface` parameter. By @fornwall in [#3905](https://github.com/gfx-rs/wgpu/pull/3905)
```

Comment:
This should be in the breaking changes section.

Reply:
Thanks for the review @cwfitzgerald !

> This should be in the breaking changes section.

It already is, or am i missing something? This is how [it looks](https://github.com/gfx-rs/wgpu/blob/41401452f6df33f306cd52437e1bd384ee2236b1/CHANGELOG.md):

<img width="1255" alt="Screenshot 2023-07-08 at 01 59 01" src="https://github.com/gfx-rs/wgpu/assets/277251/6ff334a1-1314-4a1b-a0ac-97ae4d208d84">

Reply:
Ooops I tried using the github webpage resolver and accidentally duplicated the previous changelog entry